frontend/pages/topics/[slug].tsx

```diff
@@ -28,10 +28,12 @@ interface TopicProps {
  * A page that renders the HTML of a single topic.
  */
 const TopicDetailPage: FC<TopicProps> = ({ topic }: TopicProps) => {
+  const numberOfResults = Object.keys(topic.propositions).length;
+  const topicHeader = `${topic.name} (${numberOfResults} Result${numberOfResults == 1 ? "" : "s"})`;
```

Comment:
Eh, this isn't super important in this case, but Codacy is complaining here:
![image](https://user-images.githubusercontent.com/24942155/135696677-5ac1b056-69eb-44f8-9cd4-e6e4d29b5497.png)